string object_name
---
sensor_msgs/PointCloud2 point_cloud
